magento2/usr/local/share/magento2/magento_functions.sh

```diff
@@ -202,6 +207,9 @@ function do_magento_database_create() {
 
 function do_magento_database_install() {
   set +x
+  if [ "${DATABASE_HOST}" != "localhost" ]; then
+    wait_for_remote_ports "30" "${DATABASE_HOST}:${DATABASE_PORT}"
```

Comment:
this needs https://github.com/continuouspipe/dockerfiles/pull/374 in order to not cause a regression where mysql service name is called database